### DISCUSSION TITLE
Δήλωση παρουσίας στην τακτική συνάντηση του μαθήματος

### DISCUSSION THREAD

**Discussion by epidrome** [courses-ionio/iv#14] [Announcements]
Στα παρακάτω σχόλια μπορείτε να κάνετε δήλωση παρουσίας για τις διαθέσιμες συναντήσεις που είναι κάθε 

Δευτέρα στις 1μμ ή την Παρασκευή στις 5μμ στην Αίθουσα 2.

Οι συναντήσεις γίνονται αποκλειστικά με ραντεβού μετά την έγκαιρη (δύο ή τρεις ημέρες πριν) ανάρτηση του σχολίου σας παρακάτω, έτσι ώστε να υπάρχει χρόνος να οργανώσουμε έγκαιρα τις ομάδες συνάντησης κάθε εβδομάδα.

Ο στόχος είναι να έχουμε τακτικές παραγωγικές συναντήσεις με μικρές ομάδες.

Για αυτόν τον σκοπό θα πρέπει να έρχεστε στην συνάντηση αφού πρώτα έχετε ολοκληρώσει τις πρόσφατες δραστηριότητες του προγράμματος του μαθήματος έτσι ώστε να κάνετε επίδειξη της προόδου σας. Ανάλογα με την προσέλευση και τις προτιμήσεις σας ενδέχεται να αλλάξουμε την μέρα και ώρα της συνάντησης.

Παρακαλούμε να απαντάτε στα παρακάτω σχόλια μόνο με την δήλωση παρουσίας για το αντίστοιχο σχόλιο-εβδομάδα για όποια άλλη απορία ανοίξτε νέα συζήτηση

**Comment by epidrome** [comment]
Δευτέρα 2 Οκτ. 1μμ Αίθουσα 2 (την Παρασκευή 6 Οκτ. δεν θα γίνει η συνάντηση)

| ΑΜ |
| - |
| 2003ΧΧΧ|
| |

**Comment by epidrome** [comment]
Παρασκευή 13 Οκτ. 5μμ Αίθουσα 2

| ΑΜ |
| - |
| inf2021055 |
| inf2021013 |

**Reply by Axileaszervos** [reply]
Καλησπέρα σας, θα ήθελα να δηλώσω παρουσία για το μάθημα της παρασκευής, στις 17:00.

Αχιλλέας Ζερβός/inf2021055

**Reply by nkanagno** [reply]
Καλησπέρα σας, θα ήθελα να δηλώσω παρουσία για το μάθημα της παρασκευής, στις 17:00.

Νικόλας Αναγνωστόπουλος/inf2021013

**Comment by epidrome** [comment]
~~Τρίτη 24 Οκτ. 11πμ Αίθουσα 2~~

| ΑΜ |
| - |
| 2003ΧΧΧ|
| |

**Reply by Axileaszervos** [reply]
Καλησπέρα σας, θα ήθελα να δηλώσω παρουσία για το μάθημα της τρίτης, στις 11:00.

Αχιλλέας Ζερβός/inf2021055

**Reply by epidrome** [reply]
@Axileaszervos αν δεν μπορεί και ο @nkanagno μήπως να αναβάλουμε για την επόμενη εβδομάδα την συνάντηση;

**Reply by nkanagno** [reply]
@epidrome @Axileaszervos 
Καλύτερα, για την επόμενη εβδομάδα γιατί δεν έχω προλάβει να κάνω κάποιο progress στην αναφορά.

**Reply by epidrome** [reply]
@nkanagno οκ τα λέμε την επόμενη εβδομάδα μάλλον προς τέλος ώστε να έχετε χρόνο

**Comment by epidrome** [comment]
Παρασκευή 3 Νοε. 9πμ Αίθουσα 2

| ΑΜ |
| - |
| inf2021055 |
| inf2021013 |

**Reply by Axileaszervos** [reply]
Καλημέρα σας, θα ήθελα να δηλώσω παρουσία για το μάθημα της παρασκευής, στις 9:00.

Αχιλλέας Ζερβός/inf2021055

**Reply by nkanagno** [reply]
Καλημέρα σας, θα ήθελα να δηλώσω παρουσία για το μάθημα της παρασκευής, στις 9:00.

Νικόλας αναγνωστόπουλος /inf2021013

**Comment by epidrome** [comment]
Παρασκευή 10 Νοε. 9πμ Αίθουσα 2

| ΑΜ |
| - |
| inf2021090 |
| inf2021055 |
| inf2021013 |

**Reply by nkanagno** [reply]
Καλημέρα σας, θα ήθελα να δηλώσω παρουσία για το μάθημα της παρασκευής, στις 9:00.

Νικόλας αναγνωστόπουλος /inf2021013

**Reply by Axileaszervos** [reply]
Καλημέρα σας, θα ήθελα να δηλώσω παρουσία για το μάθημα της παρασκευής, στις 9:00.

Αχιλλέας Ζερβός/inf2021055

**Reply by inf2021090** [reply]
Καλησπέρα, θα ήθελα να δηλώσω παρουσία για συμμετοχή στο μάθημα της Παρασκευής 10 Νοε. , στις 9:00.

Καφτεράνης Κωνσταντίνος/inf2021090

**Comment by epidrome** [comment]
Τρίτη 14 Νοε. 11πμ Αίθουσα 2

| ΑΜ |
| - |
| inf2021013 |
| |
| |

**Reply by nkanagno** [reply]
Καλησπέρα σας, θα ήθελα να δηλώσω παρουσία για το μάθημα της τρίτης, στις 11:00.

Νικόλας αναγνωστόπουλος /inf2021013

**Comment by epidrome** [comment]
Παρασκευή 24 Νοε. 6μμ Αίθουσα 2

| ΑΜ |
| - |
| |
| |

**Comment by epidrome** [comment]
Πέμπτη 30 Νοε. 6μμ Αίθουσα 1

| ΑΜ |
| - |
| inf2021090 |
| inf2021055 |
| inf2021013 |

**Reply by Axileaszervos** [reply]
Καλησπέρα σας @epidrome, επειδή εγώ και ο @nkanagno θα λείπουμε την παρασκευή, ειναι δυνατόν να πραγματοποιηθεί συνάντηση, νωρίτερα αυτή την εβδομάδα?

**Reply by epidrome** [reply]
Σας βολεύει μήπως κάποια ώρα την Πέμπτη το απόγευμα;

**Reply by Axileaszervos** [reply]
Ναί φυσικά.

**Reply by Axileaszervos** [reply]
Αν δεν έχει αρκετή συμμετοχή το hci , να κλείσουμε ένα ραντεβού μέσω του calendary στις ώρες διδασκαλίας του?
@epidrome

**Reply by epidrome** [reply]
ναι, αλλά είναι μόνο το πρωί της παρασκευής, 

το άλλαξα σε Πέμπτη στις 6μμ μήπως σας βολεύει καλύτερα τότε;

**Reply by Axileaszervos** [reply]
Ναι, αύριο στις 6μμ μπορούμε να παραβρεθούμε.
@epidrome

**Reply by inf2021090** [reply]
Καλησπέρα, θα ήθελα και εγώ να δηλώσω παρουσία για συμμετοχή στο μάθημα της Πέμπτης 30 Νοε. , στις 6μμ.

Καφτεράνης Κωνσταντίνος/inf2021090

**Comment by epidrome** [comment]
Παρασκευή 15 Δεκ. 6μμ Αίθουσα 2

| ΑΜ |
| - |
|  |
|  |
|  |

**Comment by epidrome** [comment]
Πέμπτη 11 Ίαν. 4μμ Αίθουσα 2
~~Παρασκευή 12 Ίαν.  9πμ Αίθουσα 2~~
~~Τρίτη 9 Ίαν.  11πμ Αίθουσα 2~~

| ΑΜ |
| - |
| inf2021090 |
| inf2021055 |
| inf2021013 |

**Reply by Axileaszervos** [reply]
Καλημέρα σας, θα ήθελα να δηλώσω παρουσία για το μάθημα της τρίτης, στις 11:00.

Αχιλλέας Ζερβός/inf2021055

**Reply by epidrome** [reply]
@Axileaszervos η συνάντηση θα γίνει την Παρασκευή γιατί δεν είχε αρκετά ραντεβού για το άλλο μάθημα, επίσης μπορείτε να κλείσετε ραντεβού όποια ώρα θέλετε Πέμπτη 3-7μμ https://calendly.com/choko/progress?month=2024-01  ελπίζω να σας βολεύει κάποια ώρα

**Comment by epidrome** [comment]
Παρασκευή 19 Ίαν. 6μμ Αίθουσα 1

| ΑΜ |
| - |
| inf2021055 |
| inf2021013 |

**Reply by nkanagno** [reply]
Καλησπέρα σας, θα ήθελα να δηλώσω παρουσία για το μάθημα της παρασκευής, στις 18:00.

Νικόλας αναγνωστόπουλος /inf2021013

**Reply by Axileaszervos** [reply]
Καλημέρα σας, θα ήθελα να δηλώσω παρουσία για το μάθημα της Παρασκευής, στις 18:00.

Αχιλλέας Ζερβός/inf2021055
